docs/guides/using-requirements-file.rst

```diff
@@ -0,0 +1,68 @@
+Using a Requirements File
```

Comment:
Because this file now goes over a pip requirements file and a conda environment file, we can update the heading to "Specifying Project Dependencies" or something generic like that